### DISCUSSION TITLE
Ability to upload a binary via CLI without running a test

### DISCUSSION THREAD

**Discussion by riglar** [devicecloud-dev/docs#4] [Ideas]
As requested here: https://discord.com/channels/1206536148650950736/1206536149254799422/1304124080995176481

**Comment by riglar** [comment]
Relevant feature delivered last week: the CLI will now detect if you've previously uploaded the given binary (via sha checksum) and won't re-upload it.

**Reply by dannyBies** [reply]
That's a very useful feature and will allow me to simplify my CICD pipelines significantly, thanks!

I do want to point out that this request is slightly different as even with this change I will be executing one extra redundant set of tests when uploading the binary for the first time.

**Comment by riglar** [comment]
Released.
